frontend/components/LiveQuery/SelectTargets.tsx

```diff
@@ -175,10 +176,10 @@ const SelectTargets = ({
   const [labels, setLabels] = useState<ILabelsByType | null>(null);
   const [searchTextHosts, setSearchTextHosts] = useState("");
   const [searchTextTeams, setSearchTextTeams] = useState<string | undefined>(
-    undefined
+    ""
```

Comment:
Nice. Will these ever be `undefined` now?

Reply:
true